ehrql/tables/beta/tpp.py

```diff
@@ -74,11 +74,11 @@ class practice_registrations(EventFrame):
                 ]
             ),
         ],
-        description=(
-            "Name of the NUTS level 1 region of England to which the practice belongs.\n"
-            "For more information see:\n"
-            "<https://www.ons.gov.uk/methodology/geography/ukgeographies/eurostat>"
-        ),
+        description=[
+            "Name of the NUTS level 1 region of England to which the practice belongs.",
+            "For more information see:",
+            "<https://www.ons.gov.uk/methodology/geography/ukgeographies/eurostat>",
+        ],
```

Comment:
Good idea!

Reply:
Actually, I've realised this would be useful for the CLI docs I'm (still) working on. Would you like me to pull this change out into a small separate PR?

Reply:
Please do -- I'm not going to get to this today

Reply:
OK, will do. Unless you object then, post merge, I'll rebase this PR and drop this initial commit.

Reply:
Thanks -- I can sort out the rebasing